sound/soc/sof/sof-pci-dev.c

```diff
@@ -284,6 +284,23 @@ static const struct sof_dev_desc jsl_desc = {
 };
 #endif
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_ALDERLAKE)
+static const struct sof_dev_desc adls_desc = {
+	.machines		= snd_soc_acpi_intel_hda_machines,
```

Comment:
I did not add any dummy acpi tables yet, but instead pointed machines to hda_machines for now. Later patches will add the ACPI machine tables if/when needed.